basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/administration/component/UserGroupMembersPanel.java

```diff
@@ -66,15 +68,56 @@ public UserGroupMembersPanel(String id, IModel<UserGroup> userGroupModel) {
 		DecoratedCoreDataTablePanel<User, UserSort> groupMemberships = 
 				DataTableBuilder.start(dataProvider, dataProvider.getSortModel())
 					.addLabelColumn(new ResourceModel("administration.usergroup.field.name"))
-						.withLink(LinkFactory.userDescriptionLinkGeneratorFactory())
-						.withClass("text text-md")
-					.addColumn(new AbstractCoreColumn<User, UserSort>(Model.of("")) {
-						private static final long serialVersionUID = 1L;
-						@Override
-						public void populateItem(Item<ICellPopulator<User>> cellItem, String componentId, IModel<User> rowModel) {
-							cellItem.add(new ActionsFragment(componentId, rowModel));
-						}
-					}).withClass("actions")
+							.withLink(LinkFactory.userDescriptionLinkGeneratorFactory())
+							.withClass("text text-md")
+					.addActionColumn()
+							.addConfirmAction(
+									new BootstrapRenderer<User>() {
+										private static final long serialVersionUID = 1L;
+										@Override
+										protected BootstrapRendererInformation doRender(User value, Locale locale) {
+											return BootstrapRendererInformation.builder()
+													.tooltip(getString("administration.usergroup.members.delete", locale))
```

Comment:
In order to shorten the code for this use case, maybe ActionRenderers could include some parameterized versions of `remove()`, `trash()` and so on, where we would only provide the resource key? Like this:
```
	public static <T> BootstrapRenderer<T> remove(final String resourceKey) {
		return new BootstrapRenderer<T>() {
			private static final long serialVersionUID = 1L;
			@Override
			protected BootstrapRendererInformation doRender(T value, Locale locale) {
				return BootstrapRendererInformation.builder()
						.label(getString(resourceKey, locale))
						.tooltip(getString(resourceKey, locale))
						.icon("fa fa-fw fa-trash-o")
						.color(BootstrapColor.DANGER)
						.build();
			}
		};
	}
```
Then this code would be much shorter.